src/pages/s.tsx

```diff
@@ -54,13 +55,14 @@ function Page(props: Props) {
         }}
       />
 
-      {/*
-        Sections: Components imported from '../components/sections' only.
-        Do not import or render components from any other folder in here.
-      */}
```

Comment:
I agree, I just do not think this was the right place for it. Maybe we should leave this only inside the `Section` component?

Reply:
I guess this comment didn't stop us from not following it when we were making our changes. But we should've paid more attention.
😆 

I like having more details about why this is needed on the documentation somewhere.

And if we're going to remove it, we should remove them altogether (there are 2 other occurrences in the Home and PDP pages).

Reply:
This was making sense to me some commits ago, where the `Section` component was being used explicitly in that file as a wrapper for other components, so the warning was coherent: It should warn us that not everything could be put inside the `Section` component, just the ones inside `components/sections` (I think).

However, some commits later I noticed that it vanished from that file (it was moved), and now the `Section` component resides inside each component it wraps (look at `Breadcrumb.tsx` for an example) - it is not being called "from outside" anymore. But the comment remains.

Reply:
@filipewl more about sections (and the `Section` component) here: https://github.com/vtex-sites/base.store/pull/371

TL;DR:
> The whole idea of sections is to split app logic into replaceable/reusable strips of screen content so that CMS users can easily replace/rearrange the desired component on the screen. (...)
> This means that the Section map has to render a self contained section. 
> (...) so CMS users can correctly change and re-order the sections

Reply:
I'll rewrite and bring back the comments, now that their purpose is clearer.